app/helpers/application_helper.rb

```diff
@@ -13,8 +13,20 @@ def divide_in_chunks(arr, chunk_size)
     chunks
   end
 
-  def get_img_url(video)
-    video.img ? "/tmp/#{video.img}" : 'http://dummyimage.com/665x375/000000/00a2ff'
+  def get_large_cover_url(video)
```

Comment:
You could also override the getter in your model to return this if the object doesn't have the image set.

```ruby
class Video
  # omitted for brevity

  def large_cover_url
    read_attribute(:large_cover_url) || 'http://dummyimage.com/665x375/000000/00a2ff'
  end
end
```

I think that should do it.

You can check [this](http://apidock.com/rails/ActiveRecord/AttributeMethods/Read/read_attribute) out for more information.